src/providers/twitch/PubSubManager.hpp

```diff
@@ -63,72 +70,59 @@ class PubSub
     std::unique_ptr<std::thread> mainThread;
```

Comment:
Before I forget this, can we rename `mainThread` to something else? I remember reading this and being really confused about this, since this isn't the main thread of the app. It's the thread the websocket client runs on.

Reply:
Resolved in [`a188976` (#5059)](https://github.com/Chatterino/chatterino2/pull/5059/commits/a1889762313be4c20321596142d6872e28e9b5d0)
I renamed it to just `thread` instead